src/accelerate/accelerator.py

```diff
@@ -223,6 +223,12 @@ def prepare_model(self, model):
                 output_device=self.local_process_index,
                 **kwargs,
             )
+        elif self.distributed_type == DistributedType.MULTI_CPU:
+            kwargs = self.ddp_handler.to_kwargs() if self.ddp_handler is not None else {}
+            model = torch.nn.parallel.DistributedDataParallel(
+                model,
+                **kwargs,
+            )
```

Comment:
```suggestion
            model = torch.nn.parallel.DistributedDataParallel(model, **kwargs)
```
This can probably fit in one line.